dashboard/app/views/levels/show.html.haml

```diff
@@ -22,6 +22,9 @@
   :javascript
     var appOptions = #{app_options.to_json};
   = render partial: 'levels/unplug', locals: {app: @game.app}
+- elsif @level.is_a?(Applab) && app_options['readonlyWorkspace'] && !app_options[:channel]
```

Comment:
perhaps it looked weird because my logic was wrong and would have totally broken the feature. Both of those fields are strings, so now I'm accessing both as strings. FWIW, `app_options` has quite a mix of string and symbol keys:
```
0> app_options.keys
=> [:levelGameName, :skinId, :baseUrl, :app, :levelId, :level, :cacheBust, :droplet, :pretty, :scriptId, :scriptName, "fullWidth", "smallFooter", "hasI18n", "callouts", "channel", "readonlyWorkspace", "isChannelBacked", "postMilestone", :applabUserId, :isAdmin, :isSignedIn, :pinWorkspaceToBottom, :hasVerticalScrollbars, :showExampleTestButtons, :rackEnv, :report, :sendToPhone, :send_to_phone_url, :copyrightStrings]
```